VTEX - Logistics API.json

```diff
@@ -4403,18 +4403,26 @@
                                         "description": "Defines the corresponding moment to the given warehouse. It is useful due to the liberation of handling order reservations.\r\n\r\nWhen set as `null`, the value will be the date/time of the request. Its format is `DateTimeOffset`, as in `yyyy-mm-dd-Thh:mm:ss`. For example: `2023-03-15T00:52:16`.",
                                         "example": "2023-03-15T00:52:16",
                                         "nullable": true
+                                    },
+                                    "leadTime": {
+                                        "type": "string",
+                                        "description": "Defines the [lead time](https://help.vtex.com/en/tutorial/lead_time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B), which is an optional time configuration you can make for a SKU in a warehouse. It can be handling time, fabrication or how long it takes for the item to be available to be shipped to customers.\r\n\r\nThe lead time is a part of the total shipping time and will be considered for shipping date calculation.\r\n\r\nThe format is `dd.hh:mm:ss` (days.hours:minutes:seconds). Note that:\r\n\r\n- Sending this field as `null` sets the value to `0`.\r\n\r\n- Not sending this field sets the value to `0`.",
+                                        "example": "10.23:59:15",
+                                        "nullable": true
                                     }
                                 },
                                 "example": {
                                     "quantity": 101,
                                     "unlimitedQuantity": false,
-                                    "dateUtcOnBalanceSystem": "2023-03-15T00:52:16"
+                                    "dateUtcOnBalanceSystem": "2023-03-15T00:52:16",
+                                    "leadTime": "10.00:00:00"
                                 }
```

Comment:
Esse exemplo não é necessário, porque já tem exemplo em cada campo do objeto e logo em seguida o exemplo completo da request. Se for remover, precisa tirar a vírgula da linha 4413 também.

Reply:
Removi o exemplo e a vírgula anterior também.